docs/com.bitwarden.organizations.md

```diff
@@ -0,0 +1,79 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden organization
+
+The `com.bitwarden.organization` doctype is used to group
+[ciphers](./com.bitwarden.ciphers.md) in a single unit and/or to share them with
+other [bitwarden contacts](./com.bitwarden.contacts.md).
+
+## Attributes
+
+- `name`: {string} - the name of the organization
+- `members`: {object} - the members of the organization (see below)
+- `defaultCollection`: {object} - the collection of the organization (see below)
+- `cozyMetadata`: {object} - the [cozyMetadata](README.md#document-metadata)
+
+## Members
+
+An organization can be shared with a list of members.
+
+- `email`: {string} - the email of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `key`: {string} - the public key of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
```

Comment:
In `com.bitwarden.contacts.md`, we refer to it as a `public_key`, I think we should stick to the same attribute name when using to the same concept

Reply:
The documentation is wrong: it is not the public key, it is the organization key encrypted with the public key of this member

Reply:
I replaced if with `the organization's key encrypted with the public key of the member from bitwarden contacts`